src/actions.yaml

```diff
@@ -18,3 +18,41 @@ db-load-datastore-config-params:
         The version number of the datastore version, e.g. 5.7.30. If not
         specified, datastore-version-name will be used as default value.
   required: [datastore, datastore-version-name, config-file]
+create-management-network:
+  description: |
+    Creates the Neutron management network for Trove.
+  params:
+    physical-network:
+      type: string
+      description: |
+        The physical network name for the flat Neutron network creation.
+        This network name must be configured in the `neutron-api` charm's
+        `flat-network-providers` config option.
+        Through this network, the Trove instances should be able to connect to
+        the RabbitMQ charm used by the OpenStack services.
+        The network can be retrieved by running:
+          openstack network list --tag charm-trove
+    cidr:
+      type: string
+      description: |
+        The subnet to be used by the Trove instances on the management network.
+    destination-cidr:
```

Comment:
It is highly likely that the trove network will be a space that will also be attached to the rabbitmq instance. This is indeed useful if the trove network offers a gateway that routes to the management network in which rabbitmq resides, but I suspect that for security reasons, most network admins will prefer that the NIC is also attached to the rabbitmq instance and will only allow egress/ingress traffic on one port (rabbitmq). 

So the trove network will most likely be created in OpenStack without a gateway, and with no additional routes. In Juju there will be a binding between rabbitmq and trove:

```bash
juju bind trove default amqp=trove-network
```

Which will result in the rabbitmq charm sending back the IP addresses it has in the trove-network space (defined in MAAS).

In MAAS we'll have to reserve a range of addresses that will never be used by MAAS to allocate to any host. That same range will be defined in OpenStack as an allocation pool. This way, MAAS and OpenStack will not clash when allocating IPs.